health-sdk/sdk/src/doc/source/event-tracking.rst

```diff
@@ -23,11 +23,30 @@ The ``GiniHealth`` class exposes kotlin flows which you can collect to track eve
      payment request.
    * ``PaymentState.Success(paymentRequest)`` when the payment request is ready and the banking app will be opened.
    * ``PaymentState.Error(throwable)`` when there was an error creating the payment request or opening the banking app.
+   * ``PaymentState.Cancel()`` when the user cancelled the payment flow.
+* ``displayedScreen`` is a ``SharedFlow`` of ``DisplayedScreen`` which emits the currently displayed screen in the ``PaymentFragment``. It can be collected to update the UI if needed, such as the toolbar title. It emits the following values:
+   * ``DisplayedScreen.Nothing`` is the default state.
+   * ``DisplayedScreen.PaymentComponentBottomSheet`` the ``PaymentComponentBottomSheet`` is displayed, showing either
+     the selected bank, or prompting the user to select one.
+   * ``DisplayedScreen.BankSelectionBottomSheet`` when the ``BankSelectionBottomSheet`` is displayed, with the list of payment providers
+     to choose from.
+   * ``DisplayedScreen.MoreInformationFragment`` when the ``MoreInformationFragment`` is displayed.
+   * ``DisplayedScreen.InstallAppBottomSheet`` when the selected payment provider is not installed.
+   * ``DisplayedScreen.OpenWithBottomSheet`` when the selected payment provider does not support GPC.
+   * ``DisplayedScreen.ShareSheet`` when the native share sheet is displayed.
+   * ``DisplayedScreen.ReviewBottomSheet`` the payment details are shown in a bottom sheet. Emitted if payment flow was started without document id.
+   * ``DisplayedScreen.ReviewFragment`` the payment details are shown in a fragment. Emitted if payment flow was started with a document id.
+* ``trustMarkersFlow`` is a ``Flow`` of ``ResultWrapper<TrustMarkerResponse>`` which emits the icons of two payment providers, along with the
+  additional payment providers count.
+   * ``ResultWrapper.Loading()`` when the payment providers are still being loaded.
+   * ``ResultWrapper.Error(throwable)`` when there was an error loading the payment providers.
+   * ``ResultWrapper.Success(trustMarkerResponse)`` when the payment providers have been loaded.
 
 PaymentComponent
```

Comment:
As discussed on Slack,  we left the flows exposed by PaymentComponent public, as they were until now, for backwards compatibility. This might change eventually.